app/qml/MainPanelButton.qml

```diff
@@ -25,10 +25,10 @@ Rectangle {
     MouseArea {
         anchors.fill: parent
         onClicked: {
-            root.activated()
+            if (!disabled) root.activated()
         }
         onPressAndHold: {
-            root.activatedOnHold()
+            if (!disabled) root.activatedOnHold()
```

Comment:
Rather than doing explicit checks for disabled property, how about we just disable the whole MouseArea when the button is disabled?